pkg/scripts/os_centos.go

```diff
@@ -111,6 +122,7 @@ func KubeadmCentOS(cluster *kubeone.KubeOneCluster, force bool) (string, error)
 	}
 
 	return Render(kubeadmCentOSTemplate, Data{
+		"LOAD_IP_TABLES":         isAWSCloudProvider(&cluster.CloudProvider),
```

Comment:
Why don't we do it regardless of the provider? I guess it shouldn't cause problems, but we can encounter other providers or images that don't have the `iptables` module enabled.